wsgi/openshift/services/models.py

```diff
@@ -1,8 +1,35 @@
 from django.db import models
+import json
+import requests
+import hashlib
 
 # Create your models here.
 
+class Location(models.Model):
+    ip = models.CharField(max_length=32, unique=True)
+    city = models.CharField(max_length=32)
+    region = models.CharField(max_length=32)
+    country = models.CharField(max_length=32)
+    latitude = models.CharField(max_length=32)
```

Comment:
I argue for keeping them as strings. I forget the name of the convention, but it is that numbers which are used for identification rather than math (such as phone numbers) should be kept in string format.